src/python/ensembl/io/genomio/integrity.py

```diff
@@ -21,211 +21,147 @@
 import hashlib
 import json
 from math import floor
-from os import path
+from os import PathLike
 from pathlib import Path
 import re
 import sys
 from typing import Any, Dict, List
 
 import argschema
 from BCBio import GFF
-from Bio import SeqIO
+from Bio import SeqIO, SeqFeature
 
 from ensembl.io.genomio.utils.archive_utils import open_gz_file
 from ensembl.io.genomio.utils.json_utils import get_json
 
 
+Lengths = Dict[str, int]
```

Comment:
`Lengths` does not seem to make the type readable to me, so at least I would suggest to add a comment above.

Reply:
Is this better?